packages/zoe/src/zoeService/startInstance.js

```diff
@@ -266,6 +266,7 @@ export const makeStartInstance = (
           contractBundleCap: newContractBundleCap,
           privateArgs: newPrivateArgs,
         };
+        state.contractBundleCap = newContractBundleCap;
```

Comment:
this is in a "feat" commit but I think it's a "fix".  Without it, `restartContract` uses the old bundleCap.

Reply:
I think the rule is "fix" for anything that's worth putting in the [changelog](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/CHANGELOG.md) under "Bug Fixes". Doesn't matter whether there's an issue or not.